serving_with_ipex_triton/start.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# Copyright (c) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache 2.0
+
+print_help() {
```

Comment:
help functions seems to be obsolete

Reply:
Edited the help function for this usecase